data/net/minecraft/world/level/levelgen/feature/OreFeature.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/world/level/levelgen/feature/OreFeature
 	METHOD canPlaceOre (Lnet/minecraft/world/level/block/state/BlockState;Ljava/util/function/Function;Ljava/util/Random;Lnet/minecraft/world/level/levelgen/feature/configurations/OreConfiguration;Lnet/minecraft/world/level/levelgen/feature/configurations/OreConfiguration$TargetBlockState;Lnet/minecraft/core/BlockPos$MutableBlockPos;)Z
 		ARG 0 state
+		ARG 1 adjacentStateAccessor
```

Comment:
As noted for `Feature#isAdjacentToAir`, this is not necessarily an accessor for _adjacent_ states but is a general accessor for position to state (which is used for checking adjacent positions).